src/use-lunatic/hooks/use-page-has-response.ts

```diff
@@ -12,15 +12,30 @@ export function usePageHasResponse(
 	executeExpression: LunaticState['executeExpression']
 ): () => boolean {
 	return useCallback(() => {
-		if (!Array.isArray(components)) {
-			return false;
+		if (!Array.isArray(components) || components.length === 0) {
+			return true;
 		}
 
+		debugger;
 		for (const component of components) {
 			// Some components are considered as "filled" by default
-			if (['PairwiseLinks', 'Roundabout'].includes(component.componentType)) {
+			if (
+				['PairwiseLinks', 'Roundabout', 'Sequence'].includes(
+					component.componentType
+				)
+			) {
+				return true;
+			}
```

Comment:
There is also subsequence that can have not input field. 

However in case of a sequence is displayed in the same page as an input component that will return a falsy `true`.

I think we should look if components has value (if no one return true) else return true only if one value is not empty